website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -9,10 +9,32 @@ description: Metrics for Consul on Kubernetes
 
 Consul on Kubernetes integrates with Prometheus and Grafana to provide metrics for Consul Service Mesh, including: service metrics, sidecar proxy metrics, Consul agent metrics, and gateway metrics. Service metrics can also be seen in the Consul UI Topology Visualization view. This section documents how to enable each of these.
 
+**Note:** Enabling the [metrics merging feature](#connect-service-and-sidecar-metrics-with-metrics-merging) with Helm value (`defaultEnableMerging`) or annotation (`consul.hashicorp.com/enable-metrics-merging`) can only be used in Consul `1.10.0-alpha1` and above. Other values to configure sidecar metrics only can still be used before Consul `1.10.0-alpha1`.
```

Comment:
It might help to specify the versions of consul-k8s and consul-helm that add support for metrics merging and the remaining metrics features.

Reply:
Yup that makes sense!